core/controllers/translator.py

```diff
@@ -204,6 +205,8 @@ def put(self, exploration_id):
             exploration_data = exp_services.get_user_exploration_data(
                 self.user_id, exploration_id)
         except:
+            # If the description of the exploration is not found for the given
```

Comment:
I don't see anywhere in this method where we're trying to find the "description of an exploration" -- that's not what exploration_data represents, I think. It's important to read and understand the code to figure out what it does represent.

Reply:
Done